src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,42 @@
 import React from 'react';
-
+import { Movie } from '../../types/MovieModel';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <>
+    <div className="card">
```

Comment:
It's better to get all needed fields before rendering HTML and then use it
```suggestion
export const MovieCard: React.FC<Props> = ({ movie }) => {
  const { title, description, imgUrl, imdbUrl } = movie;
  ...
```